apps/emqx/src/emqx_authentication.erl

```diff
@@ -27,6 +27,8 @@
 
 -include_lib("stdlib/include/ms_transform.hrl").
 
+-elvis([{elvis_style, god_modules, #{limit => 35}}]).
```

Comment:
This module requires further splitting (config handling part was moved to a new module recently), but I think this should be done in a separate PR.